spec/brainstem/dsl/field_spec.rb

```diff
@@ -99,4 +99,31 @@ def matches?(model, helper_instance, conditional_cache)
       end
     end
   end
+
+  describe "#optioned?" do
+    context "when is not an optional field" do
+      it 'returns true' do
+        expect(field.optioned?(['expensive_title', 'expensive_title2'])).to eq true
+      end
```

Comment:
Not sure how much we care here, but . . . it might be worth adding an assertion here in a before block of the context that the field we're checking is indeed not an optional field.  It took me several minutes to figure out that `expensive_title` and `expensive_title2` were established as actual fields in a separate spec helper file.